Resources/Textures/DeltaV/Mobs/Pets/lawyercarp.rsi/meta.json

```diff
@@ -0,0 +1,101 @@
+{
+  "version": 1,
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from https://github.com/tgstation/tgstation/blob/master/icons/mob/animal.dmi, edited 'dead' to create 'crit' and 'dead' states and then edited by Adeinitas on Discord to give it a suit.",
+  "states": [
```

Comment:
once you add suit as layers in sprite and mob state visualiser you can probably remove all other states here, layers can specify rsi like this
```yml
layers:
- state: alive
- sprite: DeltaV/Mobs/fishe.rsi
  state: suit
  map: ["whatever"]
```

Reply:
Where do I add the layer in the sprite and mob state visualizer, I have no idea where to put that, which is why it's gone to the meta.json.

Reply:
Wait, nevermind, I'm dumb